src/dataset/mod.rs

```diff
@@ -94,10 +95,33 @@ impl<L: Label> Label for Option<L> {}
 /// This helper struct exists to distinguish probabilities from floating points. For example SVM
 /// selects regression or classification training, based on the target type, and could not
 /// distinguish them without a new-type definition.
+///
+/// # Panics
+/// Panics if probability is negative or bigger than one.
```

Comment:
Move this to the docs of `Pr::new`, and add similar docs to `Pr::try_from` and `Pr::new_unchecked`